db/migrate/20210806104723_create_movies.rb

```diff
@@ -0,0 +1,12 @@
+class CreateMovies < ActiveRecord::Migration[6.1]
+  def change
+    create_table :movies do |t|
+      t.string :title, null: false
+      t.text :description, null: false
+      t.interval :duration, null: false
```

Comment:
I think the best type would be just `Time` 🤔

Reply:
Or even `integer` and store minutes :)

Reply:
I never used interval and would go with integer (store number of minutes), but interval should work here as well. https://www.bigbinary.com/blog/rails-6-1-adds-postgresql-interval-data-type

Reply:
> **Where there are 10 developers, there will be 11 valid solutions**

I think that all the solutions are valid, so from my perspective, it's a more personal choice ;)

Reply:
Just corrected it to integer, thank you all :)